scss/_components/_forms/_media-options.scss

```diff
@@ -61,8 +61,7 @@
     cursor: pointer;
     border: 2px solid $med-gray;
     border-radius: $sm-border-radius;
-    max-width: 100px;
-    width: 100%;
+    width: 75px;
```

Comment:
I could actually revert that change... the problem was coming from the changing border size on active class, since `border-box` sizing doesn't seem to respect `max-width`. Not an issue anymore after removing the `border-width` overrides below though.